common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -53,6 +53,8 @@ object Messages {
     namespace: Namespace,
     packageId: PackageId) extends BusMessage
 
+  final case class ImageStorageUsage(namespace: Namespace, timestamp: Instant, byteCount: Long) extends BusMessage
```

Comment:
Yeah they got changed by @rdanitz  when he added the `DeviceActivated` event. Anyway you clearly have stronger opinions  than me which one is correct so I let you decide, I just want to reach some form of consistency. Hopefully this is not something that will be changed for every commit... ;P